tests/api/driver_core.c

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Why did you create `driver_core.c`? I suppose it should appear in another pull request.

Reply:
I want to place the unittest of newly implemented function, `broadcast_buffered_txn()`, in `test_core.c`. I will seperate them once I finished them, and ensure it works fine under test.

Reply:
Create corresponding issue(s).